staging/com.smartnews/manifest.tt

```diff
@@ -0,0 +1,37 @@
+class @com.smartnews
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  list query top_articles(in opt counter: Number #[prompt="How many news articles do you want?"],
+    			  out news_id: String,
+                          out title: String,
+                          out audio: Entity(tt:url),
+                          out date: Date,
+    			  out site_name: String,
+    			  out content: String)
+  #_[confirmation="top news on SmartNews"]
```

Comment:
You must add a `#_[result]` annotation describing how the result is expressed in natural language. `#_[result]` works similarly to `#_[formatted]`, but it is text-only, and should be optimized for speech output. You can specify a single string, or you can specify multiple ones, in which case Almond will pick the most appropriate one depending on what properties the user asked for; e.g. you can have a phrase that mentions title and date, one that mentions title and author, etc. depending on whether the user asked for the date or the author.

You should also add a `#_[canonical]` form containing different phrases that refer to this query. The canonical form replaces the first primitive template you wrote (the one without any parameter).